### PR TITLE
Homepage/features

### PR DESCRIPTION
**Pull request by tammalee**
## What was done
The remainder of the home page styling. 
- The Learn about our Features section
- Jobber Resources
- Midpage CTA
- Highlight text 
- Button styling for homepage
- Some overriding of Zendesk color variables with our colors

## What wasn't done
- Footer styling was pulled out as there is a PR coming with the full footer style

## How to test

Features hover state
Screenshot 2019-12-17 10 41 00

Features focus state (tabbing through links)
Screenshot 2019-12-17 10 41 07

Jobber resources and mid-page CTA should look close to the sketch file.

Button in the head (chat now) should be ghost 2px thick border that inverts colours on hover.
Button in the mid page CTA should be primary that inverts on hover.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -212,4 +212,30 @@ document.addEventListener('DOMContentLoaded', function() {
...
+    } );
+  } );
+
+  window.addEventListener( 'scroll', () => {
+    HIGHLIGHT_TEXT.forEach( ( elm ) => {
+      if ( isInView( elm ) ) {
+        elm.classList.add( 'highlight-text--in-view' );
+      }
+    } );
+  } );
```
Comment (eddysims) [inline script.js]: This wont work in IE11. You will need to include `IE-foreach-shiv';` from marketing theme

File: styles/_base.scss [hunk 2]
```diff
@@ -32,7 +32,7 @@ body {
   }
 }
 
-h1, h2, h3, h4, h5, h6 {
+h1, h2, h3, h4, h5, h6, .h3class, .h4class {
```
Comment (eddysims) [inline styles/_base.scss]: this is an opinion: `.h3` is easier then `.h3class`

File: styles/_buttons.scss [hunk 3]
```diff
@@ -1,88 +1,69 @@
...
-  border: 1px solid $button-color;
-  border-radius: 4px;
-  box-shadow: unset;
-  color: $button-color;
-  font-size: 12px;
-  line-height: 2.34; //Taken from Zendesk garden bedrock
+  display: inline-flex;
+	position: relative;
+	align-items: center;
+	padding: 0.75rem 1.75rem;
```
Comment (eddysims) [inline styles/_buttons.scss]: tabbing issues

File: styles/_buttons.scss [hunk 4]
```diff
@@ -1,88 +1,69 @@
...
+  overflow: hidden;
+  transition: background $transition-speed, color $transition-speed, fill $transition-speed, padding $transition-speed;
```
Comment (eddysims) [inline styles/_buttons.scss]: might want to remove the stylelint disable on buttons. There are only a few errors. Most of which are new code

File: styles/_highlight-text.scss [hunk 5]
```diff
@@ -0,0 +1,47 @@
+/* stylelint-disable max-line-length */
```
Comment (eddysims) [inline styles/_highlight-text.scss]: How come this was added?

File: styles/_highlight-text.scss [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    display: block;
+  }
+
+  &--light {
+    background-image: linear-gradient(to right, transparent 50%, $highlight-light-color 50%);
+  }
+
+  &--caption {
+    background-image: linear-gradient(to right, transparent 50%, $yellow 50%);
+  }
```
Comment (eddysims) [inline styles/_highlight-text.scss]: You probably dont need all of these variants.

**Comment by eddysims** [comment]
The whole card for these should be the link. Not just the learn more.

image

File: styles/_highlight-text.scss (hunk 5 above)
Comment (tammalee) [reply styles/_highlight-text.scss]: Discussed in person

File: script.js (hunk 1 above)
Comment (tammalee) [reply script.js]: Thanks! got it!

**Comment by tammalee** [comment]
> The whole card for these should be the link. Not just the learn more.
> 
> image

Leave as is for now, we'll open another PR to change it if we go that route.

**Comment by eddysims** [comment]
This one is quite tight here. 769px.


image

File: script.js [hunk 7]
```diff
@@ -212,4 +212,31 @@ document.addEventListener('DOMContentLoaded', function() {
       })
     }
   });
+
+  //HIGHLIGHT TEXT
+
+  function isInView ( elm ) {
```
Comment (eddysims) [inline script.js]: space between function keyword and parenth

**Pull request merged**